forwards_paper.tex

```diff
@@ -97,15 +98,30 @@ \section*{Introduction}
 are all absorbed into a single compound parameter, the coalescence rate.
 These considerations, and increasing computational power, have led to a resurgence of interest in forwards-time, individual-based simulations.
 
+Another critical assumption of the coalescent is that the sample size $n$ is much smaller
```

Comment:
This is a good point, although I think that much more serious than (Kingman's coal) not approximating (other randomly mating structures) is that the closer `n` is to `N`, the more obvious deviations from random mating will be.

I think this is a good place to put it - the intro is not yet too long.

Reply:
Sounds good to me. This is definitely a topic of interest in the human genetics world, so it'd be good to get that perspective in here too.